Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -3395,6 +3396,26 @@ void WebPage::mouseEvent(const WebMouseEvent& mouseEvent, std::optional<Vector<S
     revokeSandboxExtensions(mouseEventSandboxExtensions);
 }
 
+void WebPage::performHitTestAndGetUserData(const WebMouseEvent& event, CompletionHandler<void(WebHitTestResultData&&, OptionSet<WebEventModifier>&&, UserData&&)>&& completionHandler)
```

Comment:
maybe `performHitTestForMouseEvent`?

Reply:
Renamed to `performHitTestForMouseEvent` in https://github.com/WebKit/WebKit/pull/14221/commits/c88025563e2436195514cb0c0a22883be08bf627